### PR TITLE
Refactor small things

### PR DESCRIPTION
**Pull request by youknowone**

### CODE REVIEW

File: vm/src/vm.rs [hunk 1]
```diff
@@ -764,8 +758,7 @@ impl VirtualMachine {
         code_object: &bytecode::CodeObject,
```
Comment (coolreader18) [inline vm/src/vm.rs]: Maybe we can move this to `objfunction.rs` so that we can make the fields all private?

Comment (youknowone) [reply vm/src/vm.rs]: I did it. Will you please review newly added commits?

Comment (youknowone) [reply vm/src/vm.rs]: I moved a few methods and now it looks better

Comment (coolreader18) [reply vm/src/vm.rs]: Looks good!

**Pull request merged**